src/index.html

```diff
@@ -1,9 +1,61 @@
 <html>
   <head>
     <title>VIPER</title>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="bower_components/codemirror/lib/codemirror.css">
+    <link rel="stylesheet" href="bower_components/codemirror/theme/solarized.css">
```

Comment:
1. Scrolling still occurs - vertically.
2. Buttons should be below code window, but not below visualization.
3. It is still possible to scroll horizontally a bit. 
4. When going "mobile" dimensions, visualization no longer is displayed. But it can be like this for now because I don't know what to do with it.

But it scales really good nevertheless!